enterprise/server/remote_execution/runner/runner.go

```diff
@@ -836,23 +848,46 @@ func (p *pool) Warmup(ctx context.Context) {
 	defer cancel()
 
 	eg, ctx := errgroup.WithContext(ctx)
-	for _, containerType := range executorProps.SupportedIsolationTypes {
-		containerType := containerType
-		image := platform.DefaultImage()
+	for _, cfg := range p.warmupConfigs() {
+		cfg := cfg
 		eg.Go(func() error {
-			return p.warmupImage(ctx, containerType, image)
+			return p.warmupImage(ctx, &cfg)
 		})
-		if containerType == platform.FirecrackerContainerType {
-			eg.Go(func() error {
-				return p.warmupImage(ctx, containerType, strings.TrimPrefix(hostedrunner.RunnerContainerImage, platform.DockerPrefix))
-			})
-		}
 	}
 	if err := eg.Wait(); err != nil {
 		log.Warningf("Error warming up containers: %s", err)
 	}
 }
 
+func (p *pool) warmupConfigs() []WarmupConfig {
+	var out []WarmupConfig
+	for _, isolation := range platform.GetExecutorProperties().SupportedIsolationTypes {
+		// Warm up the default execution image for all isolation types, as well
+		// as the new Ubuntu 20.04 image.
+		out = append(out, WarmupConfig{
```

Comment:
Have you looked at what the performance is like warming up 2 images? Curious how much the parallelism helps here.

Reply:
It seems like the parallelism is helping. Here are some numbers for various scenarios I tried (times are in seconds):

```
# This PR
97.0604944562
# This PR + serial image pull
127.32699358059999

# Current behavior @HEAD
74.78818619555555
# Current behavior @HEAD, + disable CI image warmup
65.98134982509998
```

This is measuring the time it takes to warmup all images -- podman pulls complete more quickly than firecracker pulls so the extra 23s added by this PR affects firecracker more than it does podman.